backends/tc/tc.def

```diff
@@ -79,61 +80,60 @@ class TCActionParam {
         std::string paramType = "";
         switch(dataType) {
             case TC::BIT_TYPE :
-                paramType += "__u" + Util::toString(bitSize);
+                absl::StrAppend(&paramType, "__u", bitSize);
                 break;
             case TC::DEV_TYPE :
-                paramType += "dev";
+                absl::StrAppend(&paramType, "dev");
                 break;
             case TC::MACADDR_TYPE :
-                paramType += "macaddr";
+                absl::StrAppend(&paramType, "macaddr");
                 break;
             case TC::IPV4_TYPE :
-                paramType += "ipv4";
+                absl::StrAppend(&paramType, "ipv4");
                 break;
             case TC::IPV6_TYPE :
-                paramType += "ipv6";
+                absl::StrAppend(&paramType, "ipv6");
                 break;
             case TC::BE16_TYPE :
-                paramType += "__be16";
+                absl::StrAppend(&paramType, "__be16");
                 break;
             case TC::BE32_TYPE :
-                paramType += "__be32";
+                absl::StrAppend(&paramType, "__be32");
                 break;
             case TC::BE64_TYPE :
-                paramType += "__be64";
+                absl::StrAppend(&paramType, "__be64");
                 break;
         } 
         return paramType;
     }
     toString {
         std::string tcActionParam = "";
-        tcActionParam += "\n\tparam ";
-        tcActionParam += paramName;
-        tcActionParam += " type ";
+        absl::StrAppend(&tcActionParam, "\n\tparam ", paramName.c_str());
+        absl::StrAppend(&tcActionParam, " type ");
```

Comment:
why multiple calls? Use a single `absl::StrCat` call and direct assignment, no need to append to empty string

Reply:
I kept multiple lines for better readability merged it to one however now.

Reply:
Then looks like there is a misunderstanding of why we are using these string routines and what are their purpose. Note that every `append` call essentially does a memory reallocation that incurs some cost. These costs might be elevated if these is used in e.g. hot loop